server/vulnerabilities/nvd/cpe_translations.json

```diff
@@ -59,5 +59,15 @@
       "product": ["7-zip"],
       "vendor": ["7-zip"]
     }
+  },
+  {
+    "software": {
+      "name": ["Docs"],
+      "source": ["chrome_extensions"]
+    },
+    "filter": {
+      "product": [],
+      "vendor": []
+    }
```

Comment:
Does this mean that for the `Docs` Chrome extension we will not look for vulnerabilities in the NVD repository because such software doesn't have a CPE?

Could we add a new field to `CPETranslationSoftware` to match only for `windows`? (this is the case here)

Reply:
Yeah - there're currently [no entries](https://nvd.nist.gov/products/cpe/search/results?namingFormat=2.3&keyword=cpe%3A2.3%3Aa%3A*%3A*%3A*%3A*%3A*%3A*%3A*%3Achrome%3A*%3A*) in the NVD dataset for the Docs Chrome extension and if in the future an entry is added we don't know yet what it will look like, so for now the best thing we can do is to skip it.

I'll add an OS field to the `CPETranslationSoftware` type and also a `Skip` field - hopefully that will clear things up.

Reply:
> I'll add an OS field to the CPETranslationSoftware type and also a Skip field - hopefully that will clear things up.

We can do this on a separate PR if it's a lot of work. This is not needed for this PR.

Reply:
> We can do this on a separate PR if it's a lot of work. This is not needed for this PR.

I'll do it on a separate PR since that will be a somewhat involved refactoring.